v1/linkml-schemas/address.yaml

```diff
@@ -36,15 +37,28 @@ classes:
 
 slots:
   uprn:
+    slot_uri: adb:uprn
```

Comment:
```suggestion
    range: integer
    slot_uri: adb:uprn
```

Reply:
uprn defined as an integer by addressbase and we are passing around as an integer based on data gathered from integration environment.

Reply:
I have a feeling this would actually be a breaking change as the address CRI is probably putting UPRN into a string in its JSON, but we could check with the Orange team.

Reply:
From the staging environment just now:

`    "credentialSubject": {
      "address": [
        {
          "addressCountry": "GB",
          "uprn": 100120012077,
          "buildingName": "",
          "streetName": "HADLEY ROAD",
          "postalCode": "BA2 5AA",
          "buildingNumber": "8",
          "addressLocality": "BATH",
          "validFrom": "2000-01-01",
          "subBuildingName": ""
        }
      ]
    }`

be good to check with the Orange team though.